hedera-node/src/main/java/com/hedera/services/state/logic/AwareNodeDiligenceScreen.java

```diff
@@ -89,7 +88,12 @@ public boolean nodeIgnoredDueDiligence(final DuplicateClassification duplicity)
 			return true;
 		}
 
-		final var payerAccountId = lookUpAccountID(accessor.getPayer());
+		final var payerLookup = aliasManager.lookUpPayerAccountID(accessor.getPayer());
+		if (payerLookup.response() != OK) {
+			txnCtx.setStatus(payerLookup.response());
+		}
+
+		final var payerAccountId = payerLookup.resolvedId();
```

Comment:
we can just use `final var payerAccountId = txnCtx.activePayer();` here and remove injecting aliasManager